src/components/Charts/ChartAdmissionsPer100k.tsx

```diff
@@ -150,7 +149,6 @@ const ChartAdmissionsPer100k: FunctionComponent<{
           regions={regions}
           width={chartWidth}
           yScale={yScale}
-          curve={curveMonotoneX}
```

Comment:
These no longer needs to be explicitly stated as a prop, since this is now a default prop.